README.md

```diff
@@ -86,6 +86,15 @@ docker exec -it <container-name> make bootstrap
 docker compose build <container-name>
 ```
 
+### Installing your local version of notifications-utils
+
+We mount your local version of utils in our containers so you can install it locally if you need to:
+
+```
+docker exec -it <container-name> bash
+pip install -e ../utils
```

Comment:
will this change be lost the next time you restart your containers?

Reply:
So I ran `make up`, then followed these instructions with my local version of utils which has some print statements in so I know when it is running.

I checked that hot code reloading works for making changes to notifications-utils

I then ctrl c'd `make up` and then ran `make up` again and my local util changes are still there.

It looks like that doesn't delete the container (you can see the containers that were running using `docker ps -a`) and it looks like running `make up` will just restart those same containers rather than creating fresh ones.

You can force docker to delete stopped containers using `docker compose rm`

Reply:
nice